### ISSUE TITLE
Light output unit recalibration?

### ISSUE THREAD

**Issue by brunosan** [wbg-bigdata/nightlights#47]
There has been several questions about the fact that the output graph goes below zero and the meaning of that. While this is due to the fact that this is related to the calibration and normalization process, may be we should consider either adding an explanation button, or recalibrate to e.g. 0-100 where 0 is the minimum across the set and 100 the maximum, or something like that.

**Comment by anandthakker** [comment]
@brunosan Yeah, I was afraid that this would end up causing confusion.  We still have pending plans to update the underlying data with cleaner outputs from Brian's stats/signal processing work, and I _think_ he might have been planning to take care of the recalibrating negative values as a part of that.  If that's the case, I'd lean towards doing the explainer for now, as a temporary stopgap

**Comment by brunosan** [comment]
We also mentioned this in India. I proposed a simple 0-100 where 0 is the minimum across the set (i.e. the value at the darkest control points across the country) and 100 is the maximum value across the set (+ some extra so next year we don't go over 100).

These was a reason not to do this, but I can't remember now. Registering here just in case.
